pom.xml

```diff
@@ -140,6 +140,11 @@
       <artifactId>xz</artifactId>
       <version>1.9</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-archiver</artifactId>
```

Comment:
This is only needed for the test case, right?
Any reason for not using `<scope>test</scope>` ?

Reply:
It's 'needed' to resolve the property - see `OutputTimestampResolver#24`, where MavenArchiver is created.
```
Date outputDate = new MavenArchiver().parseOutputTimestamp(paramValue);
```

But to be fair the functionality can be recreated, so if you don't want to add new dependency, it can be removed.

Reply:
Might be worth asking @hboutemy if the property could be exposed in the core Maven API.